rts/Lua/LuaConstEngine.cpp

```diff
@@ -31,6 +39,8 @@
  * @field buildFlags string Gets additional engine buildflags, e.g. "Debug" or "Sync-Debug"
  * @field featureSupport FeatureSupport Table containing various engine features as keys; use for cross-version compat
  * @field wordSize number Indicates the build type always 64 these days
+ * @field gameSpeed number Number of FPS
```

Comment:
Precision (FPS is more commonly about rendering but here's it's about the simulation)
```suggestion
 * @field gameSpeed number Number of simulation gameframes per second
```

Reply:
Done